src/pages/index.js

```diff
@@ -93,20 +93,20 @@ const Page = () => (
                 <h3 className="card-title mb-2 font-weight-bold">
                   <Link
                     className="homepage-headling-link"
-                    to="/pgd/latest/quickstart/"
+                    to="/pgd/latest/reference/"
                   >
-                    EDB Postgres Distributed quick start topics
+                    EDB Postgres Distributed's ready reference
                   </Link>
                 </h3>
                 <p>
-                  Use these topics to quickly evaluate and deploy EDB Postgres
-                  Distributed. They'll get you up and running with a fully
-                  configured EDB Postgres Distributed cluster using the same
-                  tools that you'll use to deploy to production.
+                  Use the new reference section in EDB Postgres Distributed to
+                  quickly look up views, catalogs, functions and variables. It's
+                  a new view of the documentation designed to speed up your
```

Comment:
I think it might read better if you switch the order of the phrases: 
> It's a new view of the documentation designed to centralize essential information and speed up your development.